de.metas.banking/de.metas.banking.base/src/main/java/de/metas/banking/payment/impl/PaySelectionBL.java

```diff
@@ -412,40 +412,4 @@ public void unlinkPaySelectionLineForBankStatement(final I_C_BankStatementLine_R
 			unlinkBankStatementLine(paySelectionLine);
 		}
 	}
-
-	@Override
```

Comment:
don't we need thid BL anymore? when implementing the doc action thingie for C_PaySelection, did you port them somewhere?

Reply:
Most part of that BL is about setting the lines as processed. 
The only relevant thing is about throwing an exception when the lines are with BankStatement
```
if (isInBankStatement(psl))
		{
			throw new AdempiereException(MSG_CannotReactivate_PaySelectionLineInBankStatement_2P, new Object[] { psl.getLine(), psl.getC_BankStatementLine().getC_BankStatement().getDocumentNo() });
		}

```

I will take that part of the code and put it in de.metas.banking.payment.PaySelectionDocumentHandler.reactivateIt(DocumentTableFields)